### ISSUE TITLE
Typo dapo readme 当中有错误

### ISSUE THREAD

**Issue by hankdhr** [volcengine/verl#1233]
dapo readme 当中有错误

seq-mean-token-sum 模式下不应该再除 torch.sum(eos_mask, dim=-1) 

```
if loss_agg_mode == "token-mean":
    pg_loss = verl_F.masked_mean(pg_losses, eos_mask)
elif loss_agg_mode == "seq-mean-token-sum":
    pg_loss = torch.sum(pg_losses * eos_mask, dim=-1) / torch.sum(eos_mask, dim=-1)
    pg_loss = torch.mean(pg_loss)
elif loss_agg_mode == "seq-mean-token-mean":
    pg_loss = torch.sum(pg_losses * eos_mask, dim=-1) / torch.sum(eos_mask, dim=-1)
    pg_loss = torch.mean(pg_loss)
else:
    raise ValueError(f"Invalid loss_agg_mode: {loss_agg_mode}")
```

**Comment by eric-haibin-lin** [comment]
Fixed now. thx for reporting
